src/Microsoft.Data.SqlClient/tests/FunctionalTests/AlwaysEncryptedTests/SqlColumnEncryptionCspProviderShould.cs

```diff
@@ -92,6 +92,7 @@ public class InvalidDecryptionParameters : DataAttribute
             private const string TCE_InvalidCspPath = "Internal error. Invalid column master key path: 'KeyName'. Use the following format for a key stored in a Microsoft cryptographic service provider (CSP): <CSP Provider Name>/<Key Identifier>.\r\nParameter name: masterKeyPath";
             private const string TCE_EmptyCspName = "Internal error. Empty Microsoft cryptographic service provider (CSP) name specified in column master key path: '/KeyName'. Use the following format for a key stored in a Microsoft cryptographic service provider (CSP): <CSP Provider Name>/<Key Identifier>.\r\nParameter name: masterKeyPath";
             private const string TCE_EmptyCspKeyId = "Internal error. Empty key identifier specified in column master key path: 'MSSQL_CSP_PROVIDER/'. Use the following format for a key stored in a Microsoft cryptographic service provider (CSP): <CSP Provider Name>/<Key Identifier>.\r\nParameter name: masterKeyPath";
+            private const string TCE_InvalidCspKeyId = "Internal error. Invalid key identifier: 'KeyName/ASKLSAVASLDJAS'. Verify that the key identifier in column master key path: 'Microsoft Enhanced RSA and AES Cryptographic Provider/KeyName/ASKLSAVASLDJAS' is valid and exists in the CSP.\r\nParameter name: masterKeyPath";
```

Comment:
Since this is invalid CSP Key test, can you use String Interpolation instead of hard-coding invalid key here?

Reply:
It is doable, I was just following the same pattern as other strings were hard-coded.